### PR TITLE
Add linkrel for topics

### PR DESCRIPTION
**Pull request by webb-ben**
Add link relation for topics with observations when publishing stations collection

### CODE REVIEW

File: .github/workflows/tests-docker.yml [hunk 1]
```diff
@@ -26,35 +26,29 @@ jobs:
         cp tests/test.env dev.env
         cat dev.env
         python3 wis2box-ctl.py config
-    - name: install loki plugin for docker
```
Comment (tomkralidis) [inline .github/workflows/tests-docker.yml:29]: Don't we still need the plugin install?

File: .github/workflows/tests-docker.yml [hunk 2]
```diff
@@ -26,35 +26,29 @@ jobs:
...
-    - name: build wis2box images and start containers ⚙️
+    - name: build wis2box
       run: |
         python3 wis2box-ctl.py build
+    - name: start containers ⚙️
+      run: |
         python3 wis2box-ctl.py start
-        sleep 5
-        python3 wis2box-ctl.py status
+        sleep 15
```
Comment (tomkralidis) [inline .github/workflows/tests-docker.yml:35]: To clarify, does this help the API / sequencing when waiting for ES?

File: .github/workflows/tests-docker.yml [hunk 3]
```diff
@@ -26,35 +26,29 @@ jobs:
...
+        python3 wis2box-ctl.py status -a
     - name: test wis2box data ingest/process/publish workflow ⚙️
       run: |
         docker exec wis2box sh -c " \
           wis2box environment create \
           && wis2box environment show \
-          && wis2box data setup --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed \
-          && wis2box data info --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed \
```
Comment (tomkralidis) [inline .github/workflows/tests-docker.yml:45]: Let's keep `wis2box data info` in the CI?  Might be valuable for debugging sane paths in the future.

File: wis2box/metadata/station.py [hunk 4]
```diff
@@ -35,6 +36,7 @@
 from wis2box.metadata.oscar import get_station_report, upload_station_metadata
 
 LOGGER = logging.getLogger(__name__)
+API_URL = 'http://wis2box-api:80/oapi'
```
Comment (tomkralidis) [inline wis2box/metadata/station.py:39]: Derive instead from `wis2box.env.API_URL`

File: wis2box/metadata/station.py [hunk 5]
```diff
@@ -60,6 +62,33 @@ def station():
     pass
 
 
+def load_topics():
```
Comment (tomkralidis) [inline wis2box/metadata/station.py:65]: Add docstring and function annotation.

File: wis2box/metadata/station.py [hunk 6]
```diff
@@ -60,6 +62,33 @@ def station():
...
+
+    dm = oaf.collection_items('discovery-metadata')
+
+    for topic in dm['features']:
+        for link in topic['properties']['associations']:
+            if link['type'] == 'OAFeat':
+                yield link
+
+
+def check_station_topics(topics, wigos_id):
```
Comment (tomkralidis) [inline wis2box/metadata/station.py:76]: Add docstring and function annotation.

File: wis2box/metadata/station.py [hunk 7]
```diff
@@ -60,6 +62,33 @@ def station():
...
+    oaf = Features(API_URL)
+
+    for topic in topics:
+        try:
+            obs = oaf.collection_items(
+                topic["title"], wigos_station_identifier=wigos_id)
```
Comment (tomkralidis) [inline wis2box/metadata/station.py:82]: `topic['id'], wigos_station_identifier=wigos_id)`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: .github/workflows/tests-docker.yml (hunk 1 above)
Comment (webb-ben) [reply .github/workflows/tests-docker.yml:29]: This happens in `python3 wis2box-ctl.py start`

File: .github/workflows/tests-docker.yml (hunk 2 above)
Comment (webb-ben) [reply .github/workflows/tests-docker.yml:35]: Yes

File: .github/workflows/tests-docker.yml (hunk 1 above)
Comment (tomkralidis) [reply .github/workflows/tests-docker.yml:29]: +1

File: wis2box/metadata/station.py (hunk 4 above)
Comment (webb-ben) [reply wis2box/metadata/station.py:39]: wis2box.env.API_URL is the URL outside of the containers... API_URL here is `http://localhost:8999/oapi` which doesn't resolve from within a container.

File: wis2box/metadata/station.py (hunk 7 above)
Comment (tomkralidis) [reply wis2box/metadata/station.py:82]: As discussed, thought you were using the entire collection JSON (vs just the link object).  Fine to keep as is then.  Suggest to rename those funcs though to `load_datasets` and `check_station_datasets`.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**